### PR TITLE
Made in Nezheda: Absolute Lunchboxes

### PR DESCRIPTION
**Pull request by ArsTeknica**
## About The Pull Request
<details>
<summary>
Adds in some  church made lunchboxes to the Theo-Mat </summary>
<hr>

<hr>
</details>

## Changelog
:cl:
Added in Absolute lunchboxes,  plus  a small selection of Lightfall rifles.
:cl:

### CODE REVIEW

**Comment by Kazkin** [comment]
Not sure how but this has map errors.

File: code/game/objects/items/weapons/storage/deferred.dm [hunk 1]
```diff
@@ -321,4 +321,43 @@
 	/obj/item/implanter/excelsior = 2,
 	/obj/item/storage/backpack = 1,
 	/obj/item/storage/backpack/satchel = 1)
+	
+	// Lunchboxes
+	
+/obj/item/storage/deferred/meatbox
+	name = "Lunchbox: Pork Katsu
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/storage/deferred.dm:328]:
```suggestion
	name = "Lunchbox: Pork Katsu"
```

**Comment by ArsTeknica** [comment]
Huh. Dunno how that happened.

**Comment by Kazkin** [comment]
So this PR has mega many problems, but I salvaged the code and hooked it into my own pr at #2622 Though I ultimately decided not to use the lunchboxes for the vendor, as I do not wish to take even less stuff away from the chef role, the boxes were kept so they could later be used for the Lemsnicate power for a future church update.

**Pull request closed without merging**